packages/account/src/Components/leave-confirm/leave-confirm.tsx

```diff
@@ -17,29 +17,25 @@ type TTransitionBlocker = RouteComponentProps & {
 };
 
 const LeaveConfirmMessage = ({ back, leave }: TLeaveConfirmMessage) => {
+    const { isMobile } = useDevice();
```

Comment:
why have we not used isDesktop here ? as isMobile will only work for Mobile view right and not tablet view